precovery/version.py

```diff
@@ -0,0 +1 @@
+__version__ = "0.1.0dev"
```

Comment:
What are your thoughts on hardcoding the version? The semantic versioning available through `setuptools_scm` seems to be the Python standard for some of the most popular packages.